cannot find package "foo"

I'm trying to use go-pry to debug this golang project-> https://github.com/komuw/meli
But go-pry is panicking with a weird error, reproduced below:

> `go-pry run *.go`
```
panic: cannot find package "foo" in any of:
	/home/komu/go/src/github.com/komuw/meli/vendor/foo (vendor tree)
	/usr/local/go/src/foo (from $GOROOT)
	/home/komu/go/src/foo (from $GOPATH)

goroutine 1 [running]:
main.InjectPry(0xc420214c00, 0x60, 0x6c8703, 0x3, 0x1, 0xc42042f6d8)
	/home/komu/go/src/github.com/d4l3k/go-pry/main.go:65 +0x14c0
main.main.func4(0xc423200740, 0x38, 0x80cde0, 0xc420231930, 0x0, 0x0, 0x0, 0x0)
	/home/komu/go/src/github.com/d4l3k/go-pry/main.go:252 +0x140
path/filepath.walk(0xc423200740, 0x38, 0x80cde0, 0xc420231930, 0xc4200ea030, 0x0, 0x0)
	/usr/local/go/src/path/filepath/path.go:356 +0x81
path/filepath.walk(0xc4233a2ea0, 0x30, 0x80cde0, 0xc420231860, 0xc4200ea030, 0x0, 0x0)
	/usr/local/go/src/path/filepath/path.go:381 +0x39a
path/filepath.walk(0xc421eb60f0, 0x26, 0x80cde0, 0xc4221a4c30, 0xc4200ea030, 0x0, 0x0)
	/usr/local/go/src/path/filepath/path.go:381 +0x39a
path/filepath.walk(0xc422661e00, 0x1e, 0x80cde0, 0xc4221a4750, 0xc4200ea030, 0x0, 0x0)
	/usr/local/go/src/path/filepath/path.go:381 +0x39a
path/filepath.walk(0xc422661da0, 0x17, 0x80cde0, 0xc4221a4680, 0xc4200ea030, 0x0, 0x0)
	/usr/local/go/src/path/filepath/path.go:381 +0x39a
path/filepath.walk(0xc4205932a0, 0x11, 0x80cde0, 0xc420426dd0, 0xc4200ea030, 0x0, 0x0)
	/usr/local/go/src/path/filepath/path.go:381 +0x39a
path/filepath.walk(0xc420216288, 0x6, 0x80cde0, 0xc420426d00, 0xc4200ea030, 0x0, 0x0)
	/usr/local/go/src/path/filepath/path.go:381 +0x39a
path/filepath.walk(0x6c8554, 0x1, 0x80cde0, 0xc4200ec0d0, 0xc4200ea030, 0x0, 0x30)
	/usr/local/go/src/path/filepath/path.go:381 +0x39a
path/filepath.Walk(0x6c8554, 0x1, 0xc4200ea030, 0x2, 0x3)
	/usr/local/go/src/path/filepath/path.go:403 +0x11d
main.main()
	/home/komu/go/src/github.com/d4l3k/go-pry/main.go:243 +0x53e
```                    

I'm using dep(https://github.com/golang/dep) as my dependency management tool

--- a/main.go
+++ b/main.go
@@ -241,7 +241,7 @@ func main() {
 	testsRequired := cmdArgs[0] == "test"
 	for _, dir := range goDirs {
 		filepath.Walk(dir, func(path string, info os.FileInfo, err error) error {
-			if !testsRequired && strings.HasSuffix(path, "_test.go") || !strings.HasSuffix(path, ".go") {
+			if !testsRequired && strings.HasSuffix(path, "_test.go") || !strings.HasSuffix(path, ".go") || strings.Contains(path, "vendor/") {
 				return nil
 			}
 			for _, file := range processedFiles {

